core/record/_record/recordReader.js

```diff
@@ -12,38 +12,28 @@ import * as Node from '../node'
 import { NodeValues } from '../nodeValues'
 
 import { keys } from './recordKeys'
-import * as NodesIndex from './recordNodesIndex'
 
 /**
  * === simple getters
  */
 export const getNodes = R.propOr({}, keys.nodes)
 
-export const getNodeByUuid = (uuid) => R.path([keys.nodes, uuid])
+export const getNodeByUuid = (uuid) => (record) => Records.getNodeByUuid(uuid)(record)
```

Comment:
```suggestion
export const getNodeByUuid = Records.getNodeByUuid
```

or even better


export { getNodeByUuid } form 'path to records ... '

Reply:
I did it on purpose to be sure about the parameters ..